### PR TITLE
Updates on largest-trades

### LINKED ISSUE #35: Double check "largest-trades" behavior when multiple trades are returned
For next week, while trading hours is running (~9:45am - 4:00pm), confirm how the `largest-trades` endpoint behaves on the following:

- How it orders the list? From largest trades to lower?
- How many elements are displayed on this list?
- Is it indeed a list or just shows the largest trade for that day? (If that is just one, the IEX documentation is wrong since it shows an array list)
- If it returns a list of elements indeed, fix tests and review our code to make sure it handles multiple trades;

cc: @dblock @gil27

### PR DESCRIPTION
**Pull request by rodolfobandeira**
closes #35 

@gil27 
@dblock

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ### 0.4.4 (next)
 
+* [#36](https://github.com/dblock/iex-ruby-client/pull/36): Updates on largest-trades - [@rodolfobandeira](https://github.com/rodolfobandeira).
```
Comment (dblock) [inline CHANGELOG.md]: Since we haven't shipped the largest trades stuff yet, no need to include this.

Comment (rodolfobandeira) [reply CHANGELOG.md]: danger-toc complaining about it.

Comment (rodolfobandeira) [reply CHANGELOG.md]: @dblock Rolled-back

**Comment by dblock** [comment]
👍

**Pull request merged**